store-api-server/src/payment/service/payment.service.ts

```diff
@@ -972,21 +975,34 @@ WHERE provider = 'simplex'
 
       events.push(event);
       for (const event1 of events) {
+        await this.createSimplexEventToDB(event1, paymentId);
         let deleteResponse = await deleteSimplexEvents(event1.event_id);
+
         if (deleteResponse?.status == 'OK') {
-          Logger.log(
-            `simplex payment DELETE event succeeded. eventId=${event1.event_id} paymentId=${paymentId}`,
-          );
+          Logger.log(`simplex payment DELETE event succeeded. eventId=${event1.event_id} paymentId=${paymentId}`);
```

Comment:
Better to not log this. We're already storing this into the database, no need to log it as well (it will only pollute the logs, ie make other logged messages less visible).